recipe/build.sh

```diff
@@ -1,36 +1,20 @@
 #!/bin/bash
+set -euxo pipefail
 
-BUILD_DIR=${SRC_DIR}/build
+BUILD_DIR="${SRC_DIR}/build"
+rm -rf "${BUILD_DIR}" || true
+mkdir -p "${BUILD_DIR}"
+cd "${BUILD_DIR}"
 
-mkdir ${BUILD_DIR}
-cd ${BUILD_DIR}
-
-PYTHON_INCLUDE_DIR=$(${PYTHON} -c 'import sysconfig;print("{0}".format(sysconfig.get_path("platinclude")))')
-PYTHON_LIBRARY=$(${PYTHON} -c 'import sysconfig;print("{0}/{1}".format(*map(sysconfig.get_config_var, ("LIBDIR", "LDLIBRARY"))))')
-
-cmake \
+cmake ${CMAKE_ARGS} \
     -G Ninja \
-    -D "CMAKE_CXX_FLAGS:STRING=-fvisibility=hidden -fvisibility-inlines-hidden ${CXXFLAGS}" \
-    -D "CMAKE_C_FLAGS:STRING=-fvisibility=hidden ${CFLAGS}" \
```

Comment:
These visibility settings can be important to reduce the size the the binary distributions. Can you report on the size of the previous release and then this one to compare the effect?